lib/node_modules/@stdlib/iter/cartesian-product/lib/main.js

```diff
@@ -0,0 +1,137 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var setReadOnly = require('@stdlib/utils/define-nonenumerable-read-only-property');
+var isCollection = require('@stdlib/assert/is-collection');
+var base = require('@stdlib/array/base/cartesian-product');
+var iteratorSymbol = require('@stdlib/symbol/iterator');
+var format = require('@stdlib/string/format');
+
+
+// MAIN //
+
+/**
+* Returns an Iterator of Cartesian product of arrays.
+*
+* @param {Collection} x1 - first input array
+* @param {Collection} x2 - second input array
+* @throws {TypeError} first argument must be a collection
+* @throws {TypeError} second argument must be a collection
+* @returns {Iterator} iterator
+*
+* @example
+* var iter = iterCartesianProduct([1, 2], [3, 4]);
+*
+* var v = iter.next().value;
+* // returns [1,3]
+*
+* v = iter.next().value;
+* // returns [1,4]
+*
+* v = iter.next().value;
+* // returns [2,3]
+*
+* v = iter.next().value;
+* // returns [2,4]
+*/
+function iterCartesianProduct(x1, x2) {
```

Comment:
```suggestion
function iterCartesianProduct( x1, x2 ) {
```

We add a space when brackets are present, please change this in all the instances of this file